src/Components/LandingPage/LandingPage.tsx

```diff
@@ -46,13 +47,18 @@ const LandingPage = ({ setFetchedScreen, handleCheckboxChange }: LandingPageProp
   });
 
   useEffect(() => {
-    errorController.updateError(formData.agreeToTermsOfService);
-  }, [formData.agreeToTermsOfService]);
+    privacyErrorController.updateError(formData.agreeToTermsOfService);
+    ageErrorController.updateError(formData.is13OrOlder);
+  }, [formData.agreeToTermsOfService, formData.is13OrOlder]);
```

Comment:
Can you split this into separate useEffects?

Reply:
Absolutely! Good call.